### ISSUE TITLE
Picasso: A Modular Framework for Visualizing the Learning Process of Neural Network Image Classifiers

### ISSUE THREAD

**Issue by richardtomsett** [dais-ita/interpretability-papers#12]
[Picasso: A Modular Framework for Visualizing the Learning Process of Neural Network Image Classifiers](https://openresearchsoftware.metajnl.com/articles/10.5334/jors.178/)
Picasso is a free open-source (Eclipse Public License) web application written in Python for rendering standard visualizations useful for analyzing convolutional neural networks. Picasso ships with occlusion maps and saliency maps, two visualizations which help reveal issues that evaluation metrics like loss and accuracy might hide: for example, learning a proxy classification task. Picasso works with the Tensorflow deep learning framework, and Keras (when the model can be loaded into the Tensorflow backend). Picasso can be used with minimal configuration by deep learning researchers and engineers alike across various neural network architectures. Adding new visualizations is simple: the user can specify their visualization code and HTML template separately from the application code.

Bibtex:

@article{henderson_rothe_2017,
title={Picasso: A Modular Framework for Visualizing the Learning Process of Neural Network Image Classifiers},
volume={5},
DOI={10.5334/jors.178},
journal={Journal of Open Research Software},
author={Henderson, Ryan and Rothe, Rasmus},
year={2017},
month={Nov}
}
